akeyless_gateway/assets/dashboards/akeyless_gateway_dashboard.json

```diff
@@ -0,0 +1,738 @@
+{
+  "title": "Akeyless GW",
+  "description": "The [Akeyless Vault Platform](www.akeyless.io) is a unified secrets management system that enables you to store, protect, rotate, and dynamically create credentials, certificates, and encryption keys.\n\n### [Official Akeyless docs](https://docs.akeyless.io/)\n### [Monitoring Akeyless With Datadog](https://docs.akeyless.io/) (cloned)",
```

Comment:
@akljph For my last comment, I was referring to these links, please update as well. Thanks!

Reply:
You should see that change in the last commit: https://github.com/DataDog/integrations-extras/pull/2062/commits/1f3155dba09f565bf3f1935644d24fb8c74e26f8

Did I do that incorrectly?

Reply:
Now I see it. Sorry about that :)

Done!